src/QsCompiler/SyntaxProcessor/CapabilityInference/Analyzer.fs

```diff
@@ -24,31 +24,22 @@ module Target =
     let create name capability =
         { _Name = name; _Capability = capability }
 
-type 'Props Pattern =
-    {
-        Capability: TargetCapability
-        Diagnose: Target -> QsCompilerDiagnostic option
-        Properties: 'Props
-    }
+type Pattern = { Capability: TargetCapability; Diagnose: Target -> QsCompilerDiagnostic option }
 
 module Pattern =
     let discard pattern =
-        {
-            Capability = pattern.Capability
-            Diagnose = pattern.Diagnose
-            Properties = ()
-        }
+        { Capability = pattern.Capability; Diagnose = pattern.Diagnose }
```

Comment:
This function can be removed too; it only acted on the Properties field, so it doesn't do anything anymore.